examples/example_dag.json

```diff
@@ -61,5 +61,5 @@
       "additional_attributes": {}
     }
   ],
-  "given_targets": {"fastqs": {"s3_dir":  "s3://idseq-samples-prod/samples/1/10/fastqs", "max_reads":75000000 } }
+  "given_targets": {"fastqs": {"s3_dir":  "s3://idseq-samples-prod/samples/1/10/fastqs", "max_fragments":75000000 } }
```

Comment:
I wonder if we should add a flag, say "count_reads". because counting seems to only applies to given target for host filtering. for alignment and post processing, it doesn't seem to matter.

Reply:
basically pass if count_reads is not set.